### ISSUE TITLE
Resizing Explorer triggers editors in split layout to default

### ISSUE THREAD

**Issue by sperka** [Microsoft/vscode#52251]
Issue Type: <b>Bug</b>

1. open a project/folder
2. open two files and split editors (two columns) while explorer open
3. resize width of the two open editors (ie move the splitter to the right)
4. try to resize explorer's width: editors will "jump" back to equal width layout

same applies when editors are split horizontally

VS Code version: Code - Insiders 1.25.0-insider (777c8e57e0eb1cbffb87a57ffd4f9c3d274f9429, 2018-06-18T05:11:30.274Z)
OS version: Darwin x64 17.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz (8 x 3100)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (1.50GB free)|
|Process Argv|/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron .|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|8.3.5
prettier-vscode|esb|1.5.0
terraform|mau|1.1.2
vscode-docker|Pet|0.0.27
remote-vscode|raf|1.1.0
java|red|0.27.0
vscode-java-debug|vsc|0.9.0
vscode-java-pack|vsc|0.3.0
vscode-java-test|vsc|0.6.1
vscode-maven|vsc|0.9.0
vscode-ansible|vsc|0.2.6

</details>

**Comment by bpasero** [comment]
I can reproduce.

**Comment by joaomoreno** [comment]
This is new, it didn't happen on yesterday's insiders.

**Comment by joaomoreno** [comment]
Nope, was already there yesterday. This came with the Alt resize behaviour refactorings. Fixing!
